cli_tools/gce_vm_image_import/importer/args_test.go

```diff
@@ -180,6 +180,13 @@ func TestTrimComputeEndpoint(t *testing.T) {
 		parseAndValidate(t, "-compute_endpoint_override", "  http://endpoint ").ComputeEndpoint)
 }
 
+func TestTrimComputeServiceAccount(t *testing.T) {
+	assert.Equal(t, "",
+		parseAndValidate(t, "-compute_service_account", " 	").ComputeServiceAccount)
+	assert.Equal(t, "default",
+		parseAndValidate(t, "-compute_service_account", " default	").ComputeServiceAccount)
```

Comment:
minor: use a non-default CE service account for testing (since default is handled in a specific way in the code)